cardano-cli/src/Cardano/CLI/EraBased/Options/Query.hs

```diff
@@ -27,6 +28,142 @@ import qualified Options.Applicative as Opt
 {- HLINT ignore "Use <$>" -}
 {- HLINT ignore "Move brackets to avoid $" -}
 
+pQueryCmdsTopLevel :: EnvCli -> Parser (QueryCmds ConwayEra)
+pQueryCmdsTopLevel envCli =
+  let parsers =
+        [ pProtocolParams envCli
+        , pTip envCli
+        , pStakePools envCli
+        , pStakeDistribution envCli
+        , pStakeAddressInfo envCli
+        , pUTxO envCli
+        , pLedgerState envCli
+        , pProtocolState envCli
+        , pStakeSnapshot envCli
+        , pPoolParams envCli
+        , pLeadershipSchedule envCli
+        , pKesPeriodInfo envCli
+        , pPoolState envCli
+        , pTxMempool envCli
+        , pSlotNumber envCli
+        ]
+      i =
+        Opt.progDesc $
+          mconcat
+            [ "Node query commands. Will query the local node whose Unix domain socket is "
```

Comment:
It would be nice to mention why this commands set is narrowed down in comparison to `conway query`.